src/server/game/Anticheat/AnticheatMgr.cpp

```diff
@@ -574,7 +574,7 @@ void AnticheatMgr::BuildReport(Player* player,uint8 reportType)
     {
         if (sConfigMgr->GetBoolDefault("Anticheat.WriteLog", true))
         {
-            TC_LOG_INFO("entities.player.character", "AnticheatMgr:: Reports reached assigned threshhold and counteracted by banning player %u (%u)", player->GetName(), player->GetGUID().ToString());
+            TC_LOG_INFO("entities.player.character", "AnticheatMgr:: Reports reached assigned threshhold and counteracted by banning player %u (%s)", player->GetName(), player->GetGUID().ToString().c_str());
```

Comment:
Hmm `%u` is still used to format `player->GetName()` which is also `std::string` https://github.com/TrinityCore/TrinityCore/blob/47de80b852389bb6e1e0c6c9eafaa86e2e6fe5af/src/server/game/Entities/Object/Object.h#L318

Reply:
I do thank you for your review, I actually thought i responded to it, but the issue was addressed.